### PR TITLE
Upgrade cognite-sdk and fix nonce updates

### PR DESCRIPTION
**Pull request by emelvarol-cognite**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cognite-transformations-cli"
-version = "2.1.2"
```
Comment (emelvarol-cognite) [inline pyproject.toml:3]: this is forgotten but this version has already a bug due to cognite-sdk, so i don't bother releasing 2.1.3 in pyPI

File: tests/test_run.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def test_watch_only(
     cli_result_list = from_table(cli_result.output)
     assert "Failed" in cli_result_list[2]
     assert "SELECT 'asd' AS externalId 'asd' AS name" == " ".join(cli_result_list[4])
-    assert "Syntax error at or near ''asd''(line 1, pos 27)" == " ".join(cli_result_list[6])
+    # assert "Syntax error at or near ''asd''(line 1, pos 27)" == " ".join(cli_result_list[6])
```
Comment (emelvarol-cognite) [inline tests/test_run.py:74]: this error is pretty random, let's not test it

File: pyproject.toml (hunk 1 above)
Comment (silvavelosa) [reply pyproject.toml:3]: we could add a pre-commit check to make sure the versions coincide (like what exists now on python sdk).... not for this change, but worth to look at it

File: tests/test_run.py (hunk 2 above)
Comment (silvavelosa) [inline tests/test_run.py:74]: hehehe.... server-side string changed again? 😅

**Review by silvavelosa — APPROVED** [review]
LGTM ✔️

File: tests/test_run.py (hunk 2 above)
Comment (emelvarol-cognite) [reply tests/test_run.py:74]: yeah don't understand why :D

File: pyproject.toml (hunk 1 above)
Comment (emelvarol-cognite) [reply pyproject.toml:3]: makes sense, I'll create a task.

Comment (emelvarol-cognite) [reply pyproject.toml:3]: https://cognitedata.atlassian.net/browse/CDF-16518

**Pull request merged**